tests/unit/src/unit/TestNullCoalescing.hx

```diff
@@ -286,6 +315,27 @@ class TestNullCoalescing extends Test {
 		eq(1, getMut());
 		eq("value", obj.field ?? "fail");
 		resetMut();
+
+		// resolve
+		var obj = new NullCoalAbstractResolve();
+		obj.field ??= "value";
+		eq(1, obj.getGetCounter());
+		eq(1, obj.getSetCounter());
+		eq("value", obj.field ?? "fail");
+
+		var value = obj.field ??= "value2";
+		eq(3, obj.getGetCounter());
+		eq(1, obj.getSetCounter());
+		eq("value", obj.field ?? "fail");
+		eq("value", value);
+
+		// TODO: this fails at the moment with some "not enough arguments error"
+		// mutAssignLeft(obj.field) ??= "not value";
```

Comment:
This fails while typing the lhs already, so the whole `??=` handling isn't even involved here. I think what happens is that it somehow tries to resolve the call while typing the macro result in `MSet` mode, or something along those lines.